x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/EsqlFunctionRegistry.java

```diff
@@ -436,7 +436,8 @@ private static FunctionDefinition[][] functions() {
                 def(UrlEncode.class, UrlEncode::new, "url_encode"),
                 def(UrlEncodeComponent.class, UrlEncodeComponent::new, "url_encode_component"),
                 def(UrlDecode.class, UrlDecode::new, "url_decode"),
-                def(Chunk.class, bi(Chunk::new), "chunk") },
+                def(Chunk.class, bi(Chunk::new), "chunk"),
+                def(TopSnippets.class, tri(TopSnippets::new), "top_snippets") },
```

Comment:
we should add this to the search functions? since it's actually receiving a query string?
the same for the docs